themes/OneNord-color-theme.json

```diff
@@ -320,7 +320,8 @@
         "name": "Comment",
         "scope": "comment",
         "settings": {
-          "foreground": "#6C7A96"
+          "foreground": "#6C7A96",
+          "fontStyle": "italic"
```

Comment:
I like to have comments in italic.

I am not sure if you guys agree with this 🤔 

Perhaps another option would be to make this setting configurable. Unfortunately I have no real experience with Javascript and Node so I am not really able to help here 😅

Reply:
I would like to leave the comments as they are. But you can additionally customize each theme for you the way you like it. [Here](https://www.alveeakand.com/how-to-modify-themes-in-vscode/) is a tutorial for that and in this case the setting would look like this:

```json
"editor.tokenColorCustomizations": {
        "[OneNord]": {
            "textMateRules": [
                {
                    "scope": "comment",
                    "settings": {
                        "fontStyle": "italic"
                    }
                }
            ]
        }
    }
```

Reply:
I have restored them to be rendered without italic 😃